db/people.yml

```diff
@@ -1139,6 +1339,23 @@ hyavas:
       group: bg
   name: Hasan Yavas
   position: Staff Scientist
+ijeong:
```

Comment:
this is my former PhD, I think he should already be in people as ikjeong...

Reply:
I did not find him in people

Reply:
Was he a PhD at Columbia?